packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -33,6 +33,39 @@ class JavascriptMessage {
 /// Callback type for handling messages sent from Javascript running in a web view.
 typedef void JavascriptMessageHandler(JavascriptMessage message);
 
+/// Information about a navigation action that is about to be executed.
+class NavigationRequest {
+  NavigationRequest({this.url, this.isMainFrame});
+
+  /// The URL a navigation is requested to.
+  final String url;
+
+  /// Whether the navigation request originated from the main HTML frame.
+  final bool isMainFrame;
+
+  @override
+  String toString() {
+    return 'NavigationRequest(url: $url, isMainFrame: $isMainFrame)';
```

Comment:
we usually use `$runtimeType` instead of hardcoding the type in the toString, it makes it work better when you subclass or rename the class or copy-paste the code

Reply:
done